yarn.lock

```diff
@@ -23222,13 +23222,13 @@ fsevents@~2.1.2:
   languageName: node
   linkType: hard
 
-"jest-runner-groups@npm:^2.1.0":
-  version: 2.1.0
-  resolution: "jest-runner-groups@npm:2.1.0"
+"jest-runner-groups@npm:2.2.0":
```

Comment:
I decided to leave the installation of `jest-runner-group@2.2.0` as-is because we shouldn't be too far behind releases if we can.